Lesson23/tugba/script.js

```diff
@@ -0,0 +1,54 @@
+const BASE_URL = 'https://api.unsplash.com';
+
+try {
```

Comment:
No need for this try - catch statement, the secrets file is added in HTML already (don't try to load it again here).